makefiles/buildtests.inc.mk

```diff
@@ -38,7 +38,7 @@ buildtest:
 	@ \
 	BUILDTESTOK=true; \
 	APP_RETRY=0; \
-	for BOARD in $$($(MAKE) -s info-boards-supported); do \
+	for BOARD in ${BOARDS}; do \
```

Comment:
(I trust your tests on this for now, I just noticed that by looking at the code ;-))